bp/src/BPConfiguration.m

```diff
@@ -78,6 +78,8 @@ typedef NS_OPTIONS(NSUInteger, BPOptionType) {
         "Number of simulators to run in parallel. (bluepill only)"},
     {'o', "output-dir", BP_MASTER | BP_SLAVE, NO, NO, required_argument, NULL, BP_VALUE | BP_PATH, "outputDirectory",
         "Directory where to put output log files (bluepill only)."},
+    {'j', "test-time-estimates-json", BP_MASTER | BP_SLAVE, NO, NO, required_argument, NULL, BP_VALUE | BP_PATH, "testTimeEstimatesJsonFile",
```

Comment:
If this is `(bluepill only)` it should be `BP_MASTER` only here. I know it's not obvious :-/

Reply:
Done. 👍